src/layer/vector/Canvas.js

```diff
@@ -138,6 +138,16 @@ export var Canvas = Renderer.extend({
 	_initPath: function (layer) {
 		this._updateDashArray(layer);
 		this._layers[Util.stamp(layer)] = layer;
+		if (layer.options.keyboard) {
+			layer._path = DomUtil.create('div', 'leaflet-canvas-accessibility-hidden', this._container);
```

Comment:
```suggestion
			layer._path = DomUtil.create('div', 'leaflet-canvas-interactive', this._container);
```

Suggesting a different class name here. Similar but different to `leaflet-interactive`. I think `accessibility` and `hidden` sounds a bit contradictive, although I realize it probably intends to mean "visually" hidden. A shorter class name is also nice.